### DISCUSSION TITLE
Mensajes

### DISCUSSION THREAD

**Discussion by M21E88** [community/community#147870] [Apps, API and Webhooks]
### Select Topic Area

General

### Body

Mensajes

**Comment by ebndev** [comment]
Hi @M21E88, thanks for being a part of the GitHub Community!

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" as GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. Any repeat and future violations may result in a temporary or indefinite block from the Community. 

We appreciate your understanding.
